src/Makefile

```diff
@@ -358,14 +358,8 @@ ifneq (,$(filter ${DEBUGAD},YES yes y Y 1))
 	DEFINES+=-DDEBUGAD
 endif
 
-ifneq (,$(filter ${DEBUG},YES yes y Y 1))
-	OSSEC_CFLAGS+=-g
-	AR_LDFLAGS+=-g
-else
-	OSSEC_CFLAGS+=-DNDEBUG
-	OFLAGS+=-O2
-	AR_LDFLAGS+=-s
-endif #DEBUG
+OSSEC_CFLAGS+=-g
+AR_LDFLAGS+=-g
 
```

Comment:
Please, explain why all optimizations are stripped.

Reply:
In the tests performed while modifying the code, it was found that by changing only `-DCMAKE_BUILD_TYPE=RelWithDebInfo`, but keeping the optimizations, no modification was made in the binaries and the compilation was practically the same as if you had `-DCMAKE_BUILD_TYPE=Release` (which is the type of compilation that is currently the default), tests were performed removing the optimizations one by one and see the result, but the only one where we had changes was with the configuration:

```
OSSEC_CFLAGS+=-g
AR_LDFLAGS+=-g
``` 
It was also compared with the resulting binaries when the compilation is done with `DEBUG=1` and it was evidenced that the size of the resulting binaries in this case are of a larger size (which is expected).